rust/feature-flags/src/flags/flag_matching_utils.rs

```diff
@@ -97,7 +94,13 @@ pub async fn fetch_and_locally_cache_all_relevant_properties(
         ),
     ];
     let conn_timer = common_metrics::timing_guard(FLAG_DB_CONNECTION_TIME, &labels);
+    let conn_acquisition_start = Instant::now();
     let mut conn = reader.as_ref().get_connection().await?;
+    let conn_acquisition_duration = conn_acquisition_start.elapsed();
+    info!(
```

Comment:
Just realized this info log is not enough. If we have connections during `get_connection` we don't make it to this log. 

For example in this [trace](https://traces.prod-us.posthog.dev/trace/1fcbc2c612461fbd0025307e5ed4de1e) we have DatabaseUnavailable error

Reply:
ooof, good catch!